types/tx/types_test.go

```diff
@@ -33,7 +32,7 @@ func (s *testMsgSuite) TestMsg() {
 
 	for _, c := range cases {
 		msg := testdata.NewTestMsg(c.signer)
-		err := tx.ValidateMsg(msg)
```

Comment:
this test was created to test `ValidateMsg`. Since we're removing that, for me this test is a bit useless now (it actually only tests `testdata.NewTestMsg#ValidateBasic`, no need to put it in `types/tx` imo)

Reply:
yeah, that makes sense. i removed it 👍🏻